pkg/reconciler/dispatcher/natsschannel_test.go

```diff
@@ -161,7 +161,7 @@ func TestFailedNatssSubscription(t *testing.T) {
 					reconciletesting.Addressable(),
 					reconciletesting.WithReady,
 					// add subscriber for channel
-					reconciletesting.WithNatssChannelSubscribers(t, "http://dummy.org"),
```

Comment:
Since you're changing, how about example.com?

Reply:
Done!